lib/northbound.c

```diff
@@ -570,8 +571,14 @@ static void nb_config_diff(const struct nb_config *config1,
 				LYD_DIFF_DEFAULTS, &diff);
 	assert(!err);
 
-	if (diff && DEBUG_MODE_CHECK(&nb_dbg_cbs_config, DEBUG_MODE_ALL))
-		nb_config_diff_dnode_log("iterating diff", diff);
+	if (diff && DEBUG_MODE_CHECK(&nb_dbg_cbs_config, DEBUG_MODE_ALL)) {
+		char *s;
+		if (!lyd_print_mem(&s, diff, LYD_JSON,
+				   LYD_PRINT_WITHSIBLINGS | LYD_PRINT_WD_ALL)) {
+			zlog_debug("nb_config_diff: %s", s);
+			free(s);
+		}
+	}
 
 	uint32_t seq = 0;
```

Comment:
again here? < i know you didn't touch it >

Reply:
done